freebasics/templates/base.html

```diff
@@ -112,7 +112,7 @@
   <!-- footer block -->
   {% block footer %}
   <div class="footer custom-template-container__no-font">
-    <p>&copy; 2016 <span id="printsitelabel2">
+    <p>&copy; 2017 <span id="printsitelabel2">
```

Comment:
@Mitso Django allows you to get the current year in the template by using {% now "Y" %} tag, so we won't need to change it every year